src/data.js

```diff
@@ -28,6 +28,16 @@ module.exports = [
       'Python',
     ],
   },
+  {
+    name: 'Matt Holovach',
+    description: 'Loves SEO, improving coding skills and good food',
+    url: 'https://www.coloradoseodesign.com/uses.php',
+    emoji: '✝',
+    country: 'us',
```

Comment:
```suggestion
    country: '🇺🇸',
```
this is the correct emoji